src/modules/jcms_migrate/jcms_migrate.install

```diff
@@ -1563,3 +1563,41 @@ function jcms_migrate_update_8150() {
     }
   }
 }
+
+/**
+ * Resave all press_package nodes for re-index.
```

Comment:
The object of the resave is to trigger re-indexing of content in other services but changes to press-packages aren't generating SNS notifications. Is this a bug or do we not require them because they are not being indexed in the search?